src/config.ts

```diff
@@ -196,6 +198,36 @@ class Config {
         return DEFAULT_SERVER;
     }
 
+    /**
+     * Sets the current network the wallet is connected to
+     */
+    setNetwork(network: string) {
+      this.NETWORK = network;
+    }
+
+    /**
+    * Gets the current network
+    *
+    * There is more than one method of setting this.
+    * The priority will be given to the network set using the config object.
+    * If not set, we look next into the storage object keys.
+    * If still not set, the default url is returned
+    */
+    getNetwork() {
```

Comment:
Missing return type

Reply:
Fixed in 58e3f03b225b28f2c427fcd5d67b8a4606b4d10b